src/css-grid/lib/grid-layout.js

```diff
@@ -1754,8 +1754,10 @@ module.exports = (function(window, document) { "use strict";
 			var infinity = 9999999.0;
 			var rowCount = this.growY ? this.rcMatrix.length : this.rcMatrix[0].length;
 			var colCount = this.growY ? this.rcMatrix[0].length : this.rcMatrix.length;
-			var fullWidth = this.element.offsetWidth - Math.max(0, rowCount - 1) * this.rowGap;
-			var fullHeight = this.element.offsetHeight - Math.max(0, colCount - 1) * this.colGap;
+			var fullWidth = this.element.offsetWidth;
+			var fullHeight = this.element.offsetHeight;
+			var fullDistributableWidth = fullWidth - Math.max(0, colCount - 1) * this.colGap;
+			var fullDistributableHeight = fullHeight - Math.max(0, rowCount - 1) * this.rowGap;
```

Comment:
I also think the case where fullHeight is 0px is supposed to be special and mean auto but I can't recall for sure.

Reply:
I see it is special-cased in `computeFlexibleTrackBreadth` but I don't see how `fullHeight` can ever be 0 when its' declaration is `var fullHeight = this.element.offsetHeight;` unless all children have 0 heights